lib/command.tag.js

```diff
@@ -0,0 +1,32 @@
+const { execSync } = require('child_process')
+const packageJson = require('../package.json')
+const { githubHandler } = require('./github')
+
+const versionPattern = /([0-9]\d*\.[0-9]\d*\.[0-9]\d*)(-alpha||-beta)?(-[0-9]\d*)?/
+
+const command = 'tag'
+const description = `Generate ${command} based on last commit message`
+const handler = async () => {
+  // array of the top 3 commits in master
+  const commits = execSync("git log --no-merges --format='%s' -n 3").toString().trim().split('\n')
+  const versionBump = commits.find((commit) => commit.startsWith('version'))
+  if (!versionBump) {
+    console.error('No version bump commit available')
+    process.exit(0)
+  }
```

Comment:
if there's no commit related to package update, workflow shouldn't run. What to use to check for that OR leave it for bash to do the check and exit the workflow with a `success` process.exit(0)?

Reply:
A few ways. gh actions has flow controls like `if` and status contexts from last step(s) like `success()` to be used in combination with these. Capturing previous step’s output (stdout) and perform a match in the next step is also an option